client/src/app/site/pages/meetings/pages/home/pages/start/components/start/start.component.html

```diff
@@ -14,25 +14,27 @@ <h2>{{ 'Home' | translate }}</h2>
 </os-head-bar>
 
 <mat-card class="spacer-bottom-60" [ngClass]="isEditing ? 'os-form-card' : 'os-card'">
-    <ng-container *ngIf="!isEditing">
-        <div class="app-content underlined-links">
```

Comment:
If you remove `app-content` you need to adjust the page object for the home page in `client/tests/integration/page-objects/meeting/home/detail.ts`. 
I would suggest adding something like `id="meeting-info-content` to the element and adjust the page object then.